tests/local_testing/test_embedding.py

```diff
@@ -1134,3 +1134,62 @@ async def test_embedding_with_extra_headers(sync_mode):
+        )
+    ]
+)
+def test_bedrock_embedding_aws_params(model, aws_params, input_data):
+    """
+    Test that AWS parameters are properly handled in request body for different model versions.
+    V1 should ignore all params, V2 should filter out aws_ params but keep model params.
+    Also tests image embedding for titan-embed-image-v1.
+    """
+    try:
+        litellm.set_verbose = True
```

Comment:
- instead make this a mock httpx_test
- can you add this instead to llm_translation folder. Make a new file called test_bedrock_embedding

Reply:
I created a new test as suggested, let me know if the test needs changes.